lib/configuration/index.js

```diff
@@ -35,9 +40,21 @@ function Config(configuration) {
   this.regex = {};
 
   // Define which file types the application willl process and display.
-  this.regex['fileOfInterest'] = new RegExp('\\.(' + filesOfInterest.join('|') + ')$');
+  this.regex['filesOfInterest'] = new RegExp('\\.(' + filesOfInterest.join('|') + ')$');
+  this.regex['excludedPaths'] = new RegExp('^(' + this.excludedPaths.join('|') + ')');
 }
 
+Config.prototype.isSupportedFile = function(path) {
+  var validFile = path.match(this.regex["filesOfInterest"]) != null;
```

Comment:
as long as there are no hyphens, keywords or leading numbers in the key name you can access members of an object with the dot notation, and that will allow you to use the regex `test` function and still look neat e.g. `this.regex.filesOfInterest.test(path)` , which will return the boolean you want.

Generally where you do want to use a comparison use the strict equality form i.e. `===` or `!==`, the non-strict forms have complex rules on implicit type coercion that are a common source of bugs.